README.md

```diff
@@ -88,6 +107,11 @@ those specified in the mapping. Each of the entries in the config provide mappin
 associated web relay input states and every RFPath defined in the sensor definition json
 file should have an entry in the preselector config. The keys in the dictionary may use the
 name of the RFPath or the index of the RFPath in the RFPaths array.
+The `sensors`, `digital_inputs`, and `analog_inputs` keys define the sensors,
+digital_inputs and analog_inputs configured on the device. Within each of the sections,
```

Comment:
Recommend checking this paragraph for consistent usage of `monospace formatting`